<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Painoindeksilaskin</class>
 <widget class="QMainWindow" name="Painoindeksilaskin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painoindeksilaskin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tulkinta</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Painoindeksi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sukupuoli      </string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QSlider" name="horizontalSlider_pituus">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Paino (kg)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QSlider" name="horizontalSlider_paino">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pituus (cm)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QSpinBox" name="spinBox_pituus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> cm</string>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QLabel" name="label_tulkinta">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLabel" name="label_bmi">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>- </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QSpinBox" name="spinBox_paino">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> kg</string>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLabel" name="label_sukupuoli">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Nainen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuToimenpide">
    <property name="title">
     <string>Toimenpide</string>
    </property>
    <addaction name="actionAlkutila"/>
    <addaction name="actionLopeta"/>
   </widget>
   <addaction name="menuToimenpide"/>
  </widget>
  <action name="actionAlkutila">
   <property name="text">
    <string>Alkutila</string>
   </property>
  </action>
  <action name="actionLopeta">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLopeta</sender>
   <signal>triggered()</signal>
   <receiver>Painoindeksilaskin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
